### DISCUSSION TITLE
recipes/breads/basic-sweet-babka/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1065] [Comments]
# recipes/breads/basic-sweet-babka/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/basic-sweet-babka/

**Comment by nicholaswilde** [comment]
2 loaves
### Cinnamon Filling
- 150 g unsalted butter , softened to room temperature
- (300g) light brown sugar (granulated sugar is ok, too)
- 2 tablespoons ground cinnamon

**Reply by nicholaswilde** [reply]
Proof at room temperature for an hour then in the fridge for an hour

**Reply by nicholaswilde** [reply]
Add salt to dough
